internal/client.go

```diff
@@ -171,16 +173,40 @@ func (c *HazelcastClient) LifecycleService() core.LifecycleService {
 	return c.lifecycleService
 }
 
+func (c *HazelcastClient) getLogLevel() int {
+	logLevel := c.properties.GetString(property.LoggingLevel)
+	logLevelInt, err := logger.GetLogLevel(logLevel)
+	if err != nil {
+		logLevelInt, _ = logger.GetLogLevel(property.LoggingLevel.DefaultValue())
+	}
+	return logLevelInt
+}
+
+func (c *HazelcastClient) initLogger() {
+	setLogger := c.Config.LoggerConfig().Logger()
+	logLevel := c.getLogLevel()
+	versionMessage := c.name + " [" + c.Config.GroupConfig().Name() + "]" + " [" + ClientVersion + "] "
+	if setLogger == nil {
+		l := logger.New()
+		l.Level = logLevel
+		l.SetVersionMessage(versionMessage)
```

Comment:
SetVersionMessage may be named better since it has mor information than the version.